wordpress-mobile/AztecEditor-iOS
# Stop converting apostrophe and quotes.

Fixes #660 

To test:
 - Open the demo app
 - Switch to HTML 
 - Check that apostrophe and quotes are not encoded.
